microsoft-edge/devtools-guide-chromium/dom/index.md

```diff
@@ -275,11 +278,11 @@ If you need to refer back to a node many times, store it as a global variable.
 
 1. Right-click `<p>The Big Sleep</p>` in the DOM Tree, and then select **Store as global variable**.  See [Appendix: Missing options](#appendix-missing-options) if the option isn't displayed.
```

Comment:
```suggestion
1. Right-click `<p>The Big Sleep</p>` in the DOM Tree, and then select **Store as global variable**.  If that option isn't displayed, see [Appendix: Missing options](#appendix-missing-options), below.
```
instead of reversed pattern:
Do A, B, and then C, if X.